scvi/inference/jvae_trainer.py

```diff
@@ -263,12 +272,23 @@ def loss(
         averaged_loss = torch.stack(losses).sum() / total_batch_size
         return averaged_loss
 
+    def _unpack_tensors(self, tensors):
+        x = tensors[_X_KEY].squeeze_(0)
+        local_l_mean = tensors[_LOCAL_L_MEAN_KEY].squeeze_(0)
+        local_l_var = tensors[_LOCAL_L_VAR_KEY].squeeze_(0)
```

Comment:
list comprehension ?

Reply:
I think better to leave it out explicitly since tensors is a dictionary and we want to explicitly show the keys.